integration/kubernetes/k8s-pod-quota.bats

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+
+setup() {
+	export KUBECONFIG=/etc/kubernetes/admin.conf
+	if sudo -E kubectl get runtimeclass | grep kata; then
+		pod_config_dir="${BATS_TEST_DIRNAME}/runtimeclass_workloads"
+	else
+		pod_config_dir="${BATS_TEST_DIRNAME}/untrusted_workloads"
+	fi
+}
+
+@test "Pod quota" {
+	resource_name="pod-quota"
+	deployment_name="deploymenttest"
+
+	# Create the resourcequota
+	sudo -E kubectl create -f "${pod_config_dir}/pod-quota.yaml"
```

Comment:
`pod-quota.yaml` is actually not defining a pod, so I think `quota.yaml` or `resource-quota.yaml` would be a better name.